代码/Reply.jsx

```
@@ -0,0 +1,185 @@
+                'Authorization':`Bearer ${token}`
+            }
+        })
+        .then(()=>{
+            console.log('发布回复成功');
+            update();
+            setReplyMap((pre)=>({...pre,[id]:''}));
+            setReplyImageMap((pre)=>({...pre,[id]:''}));
+        })
+        .catch(err=>console.log(err))
+    }
+    async function uploadQiniu(file,commentid){
```

Biome (1.9.4)

[error] Shouldn't redeclare 'uploadQiniu'. Consider to delete it or rename it.

'uploadQiniu' is defined here:


(lint/suspicious/noRedeclare)